ERROR - 2024-06-12 09:04:55 --> mysqli_sql_exception: Unknown column 'cGl0ZWtAZ21haWwuY29tMTIzNDU2' in 'field list' in C:\xampp\htdocs\crud-structured\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\crud-structured\system\Database\MySQLi\Connection.php(308): mysqli->query('UPDATE auth_use...', 0)
#1 C:\xampp\htdocs\crud-structured\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE auth_use...')
#2 C:\xampp\htdocs\crud-structured\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE auth_use...')
#3 C:\xampp\htdocs\crud-structured\app\Controllers\User.php(116): CodeIgniter\Database\BaseConnection->query('UPDATE auth_use...')
#4 C:\xampp\htdocs\crud-structured\system\CodeIgniter.php(933): App\Controllers\User->login()
#5 C:\xampp\htdocs\crud-structured\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 C:\xampp\htdocs\crud-structured\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\crud-structured\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\crud-structured\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\crud-structured\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\xampp\htdocs\crud-structured\system\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2024-06-12 09:04:55 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'cGl0ZWtAZ21haWwuY29tMTIzNDU2' in 'field list'
[Method: POST, Route: user/login]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 APPPATH\Controllers\User.php(116): CodeIgniter\Database\BaseConnection->query('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:04:55\' ,
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\User->login()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\crud-structured\\public\\index.php')
CRITICAL - 2024-06-12 09:04:55 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'cGl0ZWtAZ21haWwuY29tMTIzNDU2' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:04:55\' ,
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:04:55\' ,
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 3 APPPATH\Controllers\User.php(116): CodeIgniter\Database\BaseConnection->query('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:04:55\' ,
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\User->login()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\crud-structured\\public\\index.php')
CRITICAL - 2024-06-12 09:04:55 --> [Caused by] mysqli_sql_exception: Unknown column 'cGl0ZWtAZ21haWwuY29tMTIzNDU2' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:04:55\' ,
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:04:55\' ,
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:04:55\' ,
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 4 APPPATH\Controllers\User.php(116): CodeIgniter\Database\BaseConnection->query('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:04:55\' ,
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\User->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\crud-structured\\public\\index.php')
ERROR - 2024-06-12 09:04:56 --> mysqli_sql_exception: Unknown column 'cGl0ZWtAZ21haWwuY29tMTIzNDU2' in 'field list' in C:\xampp\htdocs\crud-structured\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\crud-structured\system\Database\MySQLi\Connection.php(308): mysqli->query('UPDATE auth_use...', 0)
#1 C:\xampp\htdocs\crud-structured\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE auth_use...')
#2 C:\xampp\htdocs\crud-structured\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE auth_use...')
#3 C:\xampp\htdocs\crud-structured\app\Controllers\User.php(116): CodeIgniter\Database\BaseConnection->query('UPDATE auth_use...')
#4 C:\xampp\htdocs\crud-structured\system\CodeIgniter.php(933): App\Controllers\User->login()
#5 C:\xampp\htdocs\crud-structured\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 C:\xampp\htdocs\crud-structured\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\crud-structured\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\crud-structured\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\crud-structured\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\xampp\htdocs\crud-structured\system\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2024-06-12 09:04:56 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'cGl0ZWtAZ21haWwuY29tMTIzNDU2' in 'field list'
[Method: POST, Route: user/login]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 APPPATH\Controllers\User.php(116): CodeIgniter\Database\BaseConnection->query('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:04:56\' ,
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\User->login()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\crud-structured\\public\\index.php')
CRITICAL - 2024-06-12 09:04:56 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'cGl0ZWtAZ21haWwuY29tMTIzNDU2' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:04:56\' ,
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:04:56\' ,
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 3 APPPATH\Controllers\User.php(116): CodeIgniter\Database\BaseConnection->query('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:04:56\' ,
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\User->login()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\crud-structured\\public\\index.php')
CRITICAL - 2024-06-12 09:04:56 --> [Caused by] mysqli_sql_exception: Unknown column 'cGl0ZWtAZ21haWwuY29tMTIzNDU2' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:04:56\' ,
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:04:56\' ,
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:04:56\' ,
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 4 APPPATH\Controllers\User.php(116): CodeIgniter\Database\BaseConnection->query('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:04:56\' ,
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\User->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\crud-structured\\public\\index.php')
ERROR - 2024-06-12 09:05:11 --> mysqli_sql_exception: Unknown column 'cGl0ZWtAZ21haWwuY29tMTIzNDU2' in 'field list' in C:\xampp\htdocs\crud-structured\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\crud-structured\system\Database\MySQLi\Connection.php(308): mysqli->query('UPDATE auth_use...', 0)
#1 C:\xampp\htdocs\crud-structured\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE auth_use...')
#2 C:\xampp\htdocs\crud-structured\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE auth_use...')
#3 C:\xampp\htdocs\crud-structured\app\Controllers\User.php(116): CodeIgniter\Database\BaseConnection->query('UPDATE auth_use...')
#4 C:\xampp\htdocs\crud-structured\system\CodeIgniter.php(933): App\Controllers\User->login()
#5 C:\xampp\htdocs\crud-structured\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 C:\xampp\htdocs\crud-structured\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\crud-structured\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\crud-structured\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 C:\xampp\htdocs\crud-structured\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 C:\xampp\htdocs\crud-structured\system\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2024-06-12 09:05:11 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'cGl0ZWtAZ21haWwuY29tMTIzNDU2' in 'field list'
[Method: POST, Route: user/login]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 APPPATH\Controllers\User.php(116): CodeIgniter\Database\BaseConnection->query('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:05:11\',
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\User->login()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\crud-structured\\public\\index.php')
CRITICAL - 2024-06-12 09:05:11 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'cGl0ZWtAZ21haWwuY29tMTIzNDU2' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:05:11\',
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:05:11\',
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 3 APPPATH\Controllers\User.php(116): CodeIgniter\Database\BaseConnection->query('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:05:11\',
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\User->login()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\crud-structured\\public\\index.php')
CRITICAL - 2024-06-12 09:05:11 --> [Caused by] mysqli_sql_exception: Unknown column 'cGl0ZWtAZ21haWwuY29tMTIzNDU2' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:05:11\',
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:05:11\',
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:05:11\',
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 4 APPPATH\Controllers\User.php(116): CodeIgniter\Database\BaseConnection->query('UPDATE auth_user SET
                    auth_user_date_login = NOW(),
                    auth_user_date_expired = \'2024-06-12 10:05:11\',
                    auth_user_token = cGl0ZWtAZ21haWwuY29tMTIzNDU2
                    WHERE auth_user_user_id = \'16\'')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\User->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\crud-structured\\public\\index.php')
CRITICAL - 2024-06-12 09:13:50 --> ErrorException: Undefined array key "Token"
[Method: GET, Route: listpublic/listproduct]
in APPPATH\Filters\Auth.php on line 45.
 1 APPPATH\Filters\Auth.php(45): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "Token"', 'C:\\xampp\\htdocs\\crud-structured\\app\\Filters\\Auth.php', 45)
 2 SYSTEMPATH\Filters\Filters.php(203): App\Filters\Auth->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 3 SYSTEMPATH\Filters\Filters.php(184): CodeIgniter\Filters\Filters->runBefore([...])
 4 SYSTEMPATH\CodeIgniter.php(481): CodeIgniter\Filters\Filters->run('listpublic/listproduct', 'before')
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\crud-structured\\public\\index.php')
CRITICAL - 2024-06-12 09:13:51 --> ErrorException: Undefined array key "Token"
[Method: GET, Route: listpublic/listproduct]
in APPPATH\Filters\Auth.php on line 45.
 1 APPPATH\Filters\Auth.php(45): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "Token"', 'C:\\xampp\\htdocs\\crud-structured\\app\\Filters\\Auth.php', 45)
 2 SYSTEMPATH\Filters\Filters.php(203): App\Filters\Auth->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 3 SYSTEMPATH\Filters\Filters.php(184): CodeIgniter\Filters\Filters->runBefore([...])
 4 SYSTEMPATH\CodeIgniter.php(481): CodeIgniter\Filters\Filters->run('listpublic/listproduct', 'before')
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\crud-structured\\public\\index.php')
CRITICAL - 2024-06-12 09:15:19 --> ErrorException: Undefined array key "Token"
[Method: GET, Route: listpublic/listproduct]
in APPPATH\Filters\Auth.php on line 42.
 1 APPPATH\Filters\Auth.php(42): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "Token"', 'C:\\xampp\\htdocs\\crud-structured\\app\\Filters\\Auth.php', 42)
 2 SYSTEMPATH\Filters\Filters.php(203): App\Filters\Auth->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 3 SYSTEMPATH\Filters\Filters.php(184): CodeIgniter\Filters\Filters->runBefore([...])
 4 SYSTEMPATH\CodeIgniter.php(481): CodeIgniter\Filters\Filters->run('listpublic/listproduct', 'before')
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\crud-structured\\public\\index.php')
CRITICAL - 2024-06-12 09:15:21 --> ErrorException: Undefined array key "Token"
[Method: GET, Route: listpublic/listproduct]
in APPPATH\Filters\Auth.php on line 42.
 1 APPPATH\Filters\Auth.php(42): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "Token"', 'C:\\xampp\\htdocs\\crud-structured\\app\\Filters\\Auth.php', 42)
 2 SYSTEMPATH\Filters\Filters.php(203): App\Filters\Auth->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 3 SYSTEMPATH\Filters\Filters.php(184): CodeIgniter\Filters\Filters->runBefore([...])
 4 SYSTEMPATH\CodeIgniter.php(481): CodeIgniter\Filters\Filters->run('listpublic/listproduct', 'before')
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\crud-structured\\public\\index.php')
CRITICAL - 2024-06-12 09:15:51 --> ErrorException: Undefined array key "Token"
[Method: GET, Route: listpublic/listproduct]
in APPPATH\Filters\Auth.php on line 42.
 1 APPPATH\Filters\Auth.php(42): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "Token"', 'C:\\xampp\\htdocs\\crud-structured\\app\\Filters\\Auth.php', 42)
 2 SYSTEMPATH\Filters\Filters.php(203): App\Filters\Auth->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
 3 SYSTEMPATH\Filters\Filters.php(184): CodeIgniter\Filters\Filters->runBefore([...])
 4 SYSTEMPATH\CodeIgniter.php(481): CodeIgniter\Filters\Filters->run('listpublic/listproduct', 'before')
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\crud-structured\\public\\index.php')
